Library/ExperimentName+HelpersTests.swift

```diff
@@ -0,0 +1,26 @@
+@testable import KsApi
+@testable import Library
+import Prelude
+import XCTest
+
+final class ExperimentName_HelpersTests: TestCase {
+  // MARK: nativeCheckout
+  func testNativeCheckoutExperimentIsEnabled() {
+
+    let config = Config.template
+      |> \.abExperiments .~ [Experiment.Name.nativeCheckoutV1.rawValue: "control"]
+
+    withEnvironment(config: config) {
+      XCTAssertFalse(nativeCheckoutExperimentIsEnabled())
+    }
+  }
+
+  func testIsEnabled_Experimental() {
+    let config = Config.template
+      |> \.abExperiments .~ [Experiment.Name.nativeCheckoutV1.rawValue: "experimental"]
```

Comment:
I'd also suggest we test when `abExperiments` is empty or does not include `nativeCheckoutV1`. What do you think?

Reply:
Good idea!